usaspending_api/references/migrations/0067_create_officers.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2017-04-13 17:41
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def update_legal_entities(apps, schema_editor):
```

Comment:
Thanks for thinking to do a data migration. I'm not sure why we need to ensure that every LegalEntity has a set of officers? If this is ensure that we have something to serialize for the front-end while we're loading in the real data, is there another way we can mock something out for them? 

Or is this to ensure that the `LegalEntity.officer` query pattern will always work? That's something I didn't think of re: the `OneToOne` pattern.